mitodl/mitxonline
# asadiqbal08/Updated the Receipt Page with additional Details

#### Pre-Flight checklist


#### What are the relevant tickets?
fixes: #531   
##### NOTE : Need to add some serializer tests.

#### What's this PR do?
As a user, I'd like to have enough information on the receipt so that I can share it with an employer for reimbursement.

#### How should this be manually tested?
- Visit the receipt page after an enrolment or by visiting `/orders/receipt/{RECEIPT_ID}/`
- Verify the required data is present as requested over the issue.


#### Screenshots (if appropriate)

![screencapture-localhost-8013-orders-receipt-2-2022-05-20-13_52_11](https://user-images.githubusercontent.com/7334669/169511833-c7d758e5-34b8-4b3b-b6b6-75e67e4418f4.png)

![screencapture-localhost-8013-orders-receipt-2-2022-05-20-15_41_51](https://user-images.githubusercontent.com/7334669/169511839-0ccdf129-85d5-44e0-9003-3e92e397130e.png)
